dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -310,6 +317,18 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         return m_useButtonStyle ? L" useButtonStyle='true'" : L"";
     }
 
+    std::wstring AppNotificationBuilder::GetProgressBars()
+    {
+        std::wstring result{};
+
```

Comment:
NIT: Remove blank line

Looks very C with variables declared in a block before code. C++ style is variables declared with code on 1st use.

Plus nowadays widescreen monitors are the common case so excess blank lines increase vertical height for less readability

Reply:
done